Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "yoga",
+    platforms: [
+        .macOS(.v10_10),
+        .iOS(.v9),
+        .tvOS(.v9)
+    ],
+    products: [
+        .library(name: "yoga", targets: [ "yoga" ])
+    ],
+    targets: [
+        .target(
+            name: "yoga",
+            path: "yoga",
+            publicHeadersPath: "include"
+        )
+    ],
+    cxxLanguageStandard: CXXLanguageStandard(rawValue: "gnu++17")
```

Comment:
Could we keep in sync with https://github.com/facebook/yoga/blob/main/Yoga.podspec?

Its min flags are roughly in sync with the range of devices that Meta is testing and using.

Reply:
Changed to `c++17`.